io.elementary.code.json

```diff
@@ -0,0 +1,247 @@
+{
+	"app-id": "io.elementary.code",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.24",
+	"sdk": "org.gnome.Sdk",
+	"command": "scratch-text-editor",
+	"rename-desktop-file": "org.pantheon.scratch.desktop",
+	"rename-appdata-file": "org.pantheon.scratch.appdata.xml",
+	"finish-args": [
+		"--filesystem=host",
+		/* Wayland and X11 access */ 
+		"--socket=wayland",
+		"--socket=x11",
+		/* own dbus name */ 
+		"--own-name=org.pantheon.scratch",
+		"--own-name=org.pantheon.terminal",
+		"--own-name=org.gnome.zeitgeist",
+		"--talk-name=org.gtk.vfs",
+		"--talk-name=org.gtk.vfs.*",
+		"--talk-name=org.gnome.SettingsDaemon",
```

Comment:
Worth noting in 3.24 this service was split out, so this might be broken or wrong. (Maybe requiring patches)